framework/quota/src/org/apache/cloudstack/quota/dao/QuotaBalanceDao.java

```diff
@@ -0,0 +1,43 @@
+//Licensed to the Apache Software Foundation (ASF) under one
+//or more contributor license agreements.  See the NOTICE file
+//distributed with this work for additional information
+//regarding copyright ownership.  The ASF licenses this file
+//to you under the Apache License, Version 2.0 (the
+//"License"); you may not use this file except in compliance
+//with the License.  You may obtain a copy of the License at
+//
+//http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing,
+//software distributed under the License is distributed on an
+//"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//KIND, either express or implied.  See the License for the
+//specific language governing permissions and limitations
+//under the License.
+package org.apache.cloudstack.quota.dao;
+
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.cloudstack.quota.vo.QuotaBalanceVO;
+
+import com.cloud.utils.db.GenericDao;
+
+public interface QuotaBalanceDao extends GenericDao<QuotaBalanceVO, Long> {
+
+    void saveQuotaBalance(List<QuotaBalanceVO> credits);
+
+    List<QuotaBalanceVO> findCreditBalance(long accountId, long domainId, Date startDate, Date endDate);
```

Comment:
Why are some ``accountId``s declared as ``long`` and some as ``Long``?  It seems that the type should be consistent across all method declarations.